test/integration/test_integration.py

```diff
@@ -689,6 +696,7 @@ def test_int8_dynamic_quant_subclass(self, device, dtype):
 
     @parameterized.expand(COMMON_DEVICE_DTYPE)
     def test_int8_weight_only_quant_subclass(self, device, dtype):
+        undo_recommended_configs()
```

Comment:
why do we need these?

Reply:
a bunch of resource usage errors

https://github.com/pytorch/ao/actions/runs/9657170605/job/26635939215